content/blog/contributor-highlight-daniel-chambre/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Contributor Spotlight: Daniel Chambre"
+date: 2025-04-25T14:51:43+02:00
+draft: false
+image:
+    url: daniel.jpg
+    attribution:
+author: fdrees
+tags:
+ - kubernetes
+ - service-mesh
+ - dba
+ - postgresql
+ - open-source
+summary: "In a mini-series on this blog we highlight the work of the community.
+  Today we meet Daniel Chambre (aka smiyc), a DBA at WienIT."
+---
+
+Building and maintaining and open source project takes a village. In a
+mini-series on this blog we would like to highlight the work of our
+maintainers, component owners, and members of the larger community.
+
+
+Today we're meeting Daniel Chambre, better known under his handle: 
+[smiyc](https://github.com/smiyc/). Daniel is a DBA at WienIT, working part 
+remotely from "the outskirts of Vienna". He first learned about CNP by/for EDB 
+in late 2021, and then CloudNativePG at the time it was released. 
+
+Daniel first got involved with the project by filing an Issue, that resulted 
+in a discussion, in November 2022. He's been active in the project and the 
```

Comment:
"An Issue that resulted in a discussion."
This is Github-speak. If someone is not familiar with the GH terminology, it will sound loopy.
IMO would be better to talk about the topic of discussion.